model/todo.go

```diff
@@ -0,0 +1,22 @@
+package model
+
+import (
+	"time"
+)
+
+// TODOを保持するモデル
```

Comment:
comment on exported type Todo should be of the form "Todo ..." (with optional leading article)